src/components/Gallery/Gallery.js

```diff
@@ -175,9 +175,11 @@ class Gallery extends Component {
   render() {
     const { index, exitLeft, exitRight, closing, focus } = this.state
     const { onClose, items } = this.props
-    const { src, caption, credit, alt } = items[index]
+    const { src, caption, credit } = items[index]
```

Comment:
`const currentItem = items[index]` would lead to more readable code down below.

Reply:
I agree.